lib/trino-filesystem/src/test/java/io/trino/filesystem/AbstractTestTrinoFileSystem.java

```diff
@@ -483,6 +483,14 @@ void testOutputFile()
 
                 // verify nothing changed
                 assertThat(tempBlob.read()).isEqualTo("initial");
+
```

Comment:
We could rename `supportsCreateWithoutOverwrite()` method to `supportsExclusiveCreate()`